Makefile

```diff
@@ -400,6 +400,21 @@ install-tools:
 	$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.1
 	$(GO) install mvdan.cc/gofumpt@latest
 
+.PHONY: install-test-tools
+install-test-tools:
+    $(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.1
+
+.PHONY: install-build-tools
+install-build-tools:
+    $(GO) install mvdan.cc/gofumpt@latest
```

Comment:
fumpt runs as linter, so needs to be in test-tools

build-tools would be `goversioninfo`